server/services/oneThousandCalculationsService.ts

```diff
@@ -265,6 +303,28 @@ export default class OneThousandCalculationsService {
     return ''
   }
 
+  isHdced14DayRule(breakdown: CalculationBreakdown, calc: BookingCalculation): 'Y' | 'N' | '' {
+    if (calc?.dates.HDCED) {
+      if (breakdown?.breakdownByReleaseDateType?.HDCED?.rules?.includes('HDCED_MINIMUM_14D')) {
+        return 'Y'
+      } else {
+        return 'N'
+      }
+    }
+    return ''
+  }
+
+  hasPcscSdsPlus(questions: CalculationUserQuestions): 'Y' | 'N' | '' {
+    if (questions) {
```

Comment:
could use ternary operator to replace 319 to 323
`return questions.sentenceQuestions ? 'Y' : 'N'`